### ISSUE TITLE
No user definitions in configs

### ISSUE THREAD

**Issue by NathanDotTo** [CiscoDevNet/opendaylight-setup#3]
Hi Giles

There do not seem to be any users defined in the configurations for the XRv devices in the 5 node network. Does that mean you are relying on a default cisco/cisco user?

Thanks

Nathan

**Comment by gilesheron** [comment]
Yup - the admin account gets added when you boot the node for the first time (in the expect script).

On 30 Aug 2016, at 07:57, Nathan John Sowatskey <notifications@github.com<mailto:notifications@github.com>> wrote:

Hi Giles

There do not seem to be any users defined in the configurations for the XRv devices in the 5 node network. Does that mean you are relying on a default cisco/cisco user?

Thanks

Nathan

##
